From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Subject: [PATCH] Fix installation of manual pages
Date: Mon, 14 Nov 2022 08:12:40 +0100

Use "install" instead, to setup permissions and to avoid touching the
permissions of files of unrelated packages.

Upstream-Status: Unknown
Origin: Source Mage GNU/Linux
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---

diff --git a/man/Makefile b/man/Makefile
index 26b54830a14c..1f50030a9a1d 100644
--- a/man/Makefile
+++ b/man/Makefile
@@ -3,11 +3,6 @@ clean:
 
 install:
 	@echo "Man pages (MANDIR) being installed in: \"$(MANDIR)\""; 
-	-mkdir -p $(MANDIR)/man3
-	-chmod go+rx $(MANDIR)/man3
-	-cp man3/PAPI*.3 $(MANDIR)/man3
-	-chmod go+r $(MANDIR)/man3/PAPI*.3
-	-mkdir -p $(MANDIR)/man1
-	-chmod go+rx $(MANDIR)/man1
-	-cp man1/*.1 $(MANDIR)/man1
-	-chmod go+r $(MANDIR)/man1/*.1
+	install -m755 -d $(MANDIR)/man3 $(MANDIR)/man1
+	install -m444 man3/PAPI*.3 $(MANDIR)/man3
+	install -m444 man1/*.1 $(MANDIR)/man1
